sanic/app.py

```diff
@@ -658,9 +662,10 @@ def _helper(self, host="127.0.0.1", port=8000, debug=False,
             server_settings['run_async'] = True
 
         # Serve
-        proto = "http"
-        if ssl is not None:
-            proto = "https"
-        log.info('Goin\' Fast @ {}://{}:{}'.format(proto, host, port))
+        if host and port:
```

Comment:
This will always be true because they have default values `"127.0.0.1"` and `8000`